### PR TITLE
add iframe snipet

### PR DESCRIPTION
**Pull request by iwk7273**
I have created an article about JS snippets that click on elements in an Iframe. Please review this one.

created by iwakami (CSpE)

### CODE REVIEW

**Comment by ozakimami** [comment]
@harokki 
Could you review the snippet??

File: content/en/snippets/others/click_element_in_iframe.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+```js
+var selectorIframeHtml = "<TODO: REPLACE>";
+var selectorTargetElement = "<TODO: REPLACE>";
+
+/**
+ * --------- Anything below this does not need to be changed. ---------------
+ */
+
+/**
+ * Locate the document in iframe
```
Comment (harokki) [inline content/en/snippets/others/click_element_in_iframe.md:18]: [nits] I think the element that can be taken at this point is [HTMLIFrameElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement), so "Locate the iframe" seems to be better.

**Review by harokki — CHANGES_REQUESTED** [review]

File: content/en/snippets/others/click_element_in_iframe.md (hunk 1 above)
Comment (harokki) [reply content/en/snippets/others/click_element_in_iframe.md]: @iwk7273 Thank you! Could you edit Japanese version as well? 🙏

Comment (iwk7273) [reply content/en/snippets/others/click_element_in_iframe.md]: @harokki I forgot to edit ja varsion...
I modified the comment  "iframe内のdocumentを取得する" to " iframeを探す", is it Ok for you?

Comment (harokki) [reply content/en/snippets/others/click_element_in_iframe.md]: @iwk7273 
Thanks! LGTM!

**Review by harokki — APPROVED** [review]

**Review by otoyo — APPROVED** [review]
LGTM but a script can only access a `contentWindow` property when both pages belong to the same origin so I guess it is better to note that.
FYI: https://developer.mozilla.org/ja/docs/Web/HTML/Element/iframe#%E3%82%B9%E3%82%AF%E3%83%AA%E3%83%97%E3%83%88%E6%93%8D%E4%BD%9C

**Comment by iwk7273** [comment]
> LGTM but a script can only access a `contentWindow` property when both pages belong to the same origin so I guess it is better to note that. FYI: https://developer.mozilla.org/ja/docs/Web/HTML/Element/iframe#%E3%82%B9%E3%82%AF%E3%83%AA%E3%83%97%E3%83%88%E6%93%8D%E4%BD%9C

@otoyo Thank you for the review!
I will create a FAQ about the problems that occur with cross-origin.

**Pull request merged**